src/parser/parser_support.c

```diff
@@ -4458,6 +4460,12 @@ pt_limit_to_numbering_expr (PARSER_CONTEXT * parser, PT_NODE * limit, PT_OP_TYPE
       sum->data_type->info.data_type.precision = DB_MAX_NUMERIC_PRECISION;
       sum->data_type->info.data_type.dec_precision = 0;
 
+      if (oracle_style_divide)
+	{
+	  /* It is necessary to prepare for the division operation in advance. */
+	  sum->data_type->info.data_type.dec_precision = DB_DEFAULT_NUMERIC_DIVISION_SCALE;	/* 9 */
+	}
+
```

Comment:
```suggestion
      sum->data_type->info.data_type.dec_precision = 0;

      if (oracle_style_divide)
	{
	  /* It is necessary to prepare for the division operation in advance. */
	  sum->data_type->info.data_type.dec_precision = DB_DEFAULT_NUMERIC_DIVISION_SCALE;	/* 9 */
	}

```
      sum->data_type->info.data_type.dec_precision = oracle_style_divide ? DB_DEFAULT_NUMERIC_DIVISION_SCALE : 0;

Reply:
I'd like to keep for readability